# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/blender/compile.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

This works around some Makefile limitations I want not invest time in fixing
correctly ...

  - Rene Rebe <rene@rocklinux.org>

diff -ur blender-2.33-orig/source/Makefile blender-2.33/source/Makefile
--- blender-2.33-orig/source/Makefile	2004-04-16 17:55:12.000000000 +0200
+++ blender-2.33/source/Makefile	2004-05-13 13:24:46.189355464 +0200
@@ -107,10 +107,10 @@
     COMLIB += $(OCGDIR)/gameengine/blconverter/$(DEBUG_DIR)libblconverter.a
     COMLIB += $(OCGDIR)/gameengine/blphys/common/$(DEBUG_DIR)libcommon.a
     COMLIB += $(OCGDIR)/gameengine/ketsji/$(DEBUG_DIR)libketsji.a
-    COMLIB += $(NAN_SOLID)/lib/libsolid.a
+    COMLIB += $(SRCHOME)/../extern/solid/src/.libs/libsolid.a
     COMLIB += $(OCGDIR)/gameengine/blphys/sumo/$(DEBUG_DIR)libsumo.a
     COMLIB += $(OCGDIR)/gameengine/blphys/fuzzics/$(DEBUG_DIR)libfuzzics.a
-    COMLIB += $(NAN_QHULL)/lib/libqhull.a
+    COMLIB += $(SRCHOME)/../extern/qhull/src/libqhull.a
    COMLIB += $(OCGDIR)/gameengine/blphys/dummy/$(DEBUG_DIR)libdummy.a
     COMLIB += $(OCGDIR)/gameengine/blphys/common/$(DEBUG_DIR)libcommon.a
 #    COMLIB += $(OCGDIR)/gameengine/blphys/sumo/$(DEBUG_DIR)libsumo.a
@@ -316,11 +316,10 @@
     BINTARGETS += xplink
     BINTARGETS += blenderdynamic
     BINTARGETS += blenderplayer
-    BINTARGETS += blenderstatic
   endif
   ifeq ($(CPU),powerpc)
     BINTARGETS = blenderdynamic
-    BINTARGETS += blenderstatic
+    BINTARGETS += blenderplayer
   endif
     PYLIB = $(NAN_PYTHON)/lib/python$(NAN_PYTHON_VERSION)/config/libpython$(NAN_PYTHON_VERSION).a
     PYLIB += $(NAN_MXTEXTTOOLS)
diff -ur blender-2.33-orig/source/nan_definitions.mk blender-2.33/source/nan_definitions.mk
--- blender-2.33-orig/source/nan_definitions.mk	2004-05-01 20:07:58.000000000 +0200
+++ blender-2.33/source/nan_definitions.mk	2004-05-13 13:20:58.566959328 +0200
@@ -276,7 +276,7 @@
     export NAN_FREETYPE ?= /usr
     export NAN_GETTEXT ?= /usr
     export NAN_SDL ?= $(shell sdl-config --prefix)
-    export NAN_SDLLIBS ?= $(shell sdl-config --libs)
+    export NAN_SDLLIBS ?= $(shell sdl-config --libs) -lsmpeg
     export NAN_SDLCFLAGS ?= $(shell sdl-config --cflags)
 
     # Uncomment the following line to use Mozilla inplace of netscape
